src/Kernel/CompiledMethod.class.st

```diff
@@ -1116,12 +1116,6 @@ CompiledMethod >> tags [
 	^{protocol}
 ]
 
-{ #category : #'source code management' }
```

Comment:
Can you explain why tempNames are removed?

Reply:
It was not removed, it was pulled up in CompiledCode.
This is because compiled blocks may also use it  for certain purpose such as tooling !
Such as in this case

Reply:
However this is worth a fix in the comment